src/components/PostcodeLookup/PostcodeLookup.js

```diff
@@ -350,26 +350,26 @@ class PostcodeLookup extends Component {
    * @return {*}
    */
   render() {
+    const isBrowser = browser();
     const postCodeField = {
       id: 'postcode',
       type: 'text',
       placeholder: 'SE1 7TP',
       buttonText: 'FIND ADDRESS',
-      pattern: /^[A-Za-z0-9 \\,./&-]{1,15}$/,
+      pattern: /^[A-Za-z0-9]+[A-Za-z0-9 \\,./&-]{1,15}$/,
       invalidErrorText: 'Please enter a valid postcode',
       emptyFieldErrorText: 'Please enter your postcode',
       extraClass: 'search-box',
-      autocomplete: 'off',
+      autocomplete: isBrowser.name === 'chrome' ? 'false' : 'off',
```

Comment:
hmm hey I'm not sure if this is working as expected. When I'm testing in Chrome it still fills in my postcode automatically with autofill. Did it work for you?
Also have you had a look into whether using "postcode" as a value might solve the wrongly filled in fields? 
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#Values
Would be good to test this storybook branch in a giftaid environment.

Reply:
I believe Chrome requires the form to have the attribute `autocomplete=off` and the input `autocomplete=false`

Reply:
yes, I see, but did it actually work for you in Chrome? I might be doing something wrong here.

Reply:
haven't tested in chrome

Reply:
ok ;)